docs/pages/tutorials/minimal/use-onchain.mdx

```diff
@@ -0,0 +1,255 @@
+# Use MUD from onchain code
+
+In this tutorial you learn how to use a `World` and the systems in it from code that runs on the blockchain.
+You can use this knowledge to add sub-systems to a `World` or to use MUD as part of a bigger dapp.
+
+## Setup
+
+[Create a minimal application](../minimal) and run it.
+
+## Use the `World` from Foundry
```

Comment:
I feel like a much easier way to interact with a MUD contract from the CLI would be to write a simple forge script (https://book.getfoundry.sh/reference/forge/forge-script) and execute it via the CLI. The actual code would be similar to the functions you're executing via `cast` here, but it would be much less cluttered. 

The thing I want to avoid is that people read this and think it's very complicated to interact with MUD contracts, while there would be a much easier to understand recommended method.

Another thing we can do would be to add a simple CLI command to the MUD cli to read table values that automatically requests the table's schema and handles the encoding/decoding.